pyproject.toml

```diff
@@ -42,6 +43,8 @@ test = ["coverage", "pexpect", "wheel", "ruff", "mypy"]
 "Issue Tracker"= "https://github.com/kislyuk/argcomplete/issues"
 "Change Log"= "https://github.com/kislyuk/argcomplete/blob/master/Changes.rst"
 
+[tool.setuptools_scm]
```

Comment:
Described way from https://pypi.org/project/setuptools-scm/ .
Would this be alright @kislyuk or should I see if it can be moved to to setup.py?

Reply:
This is fine.